### DISCUSSION TITLE
Submit your site power by Geeky Hugo

### DISCUSSION THREAD

**Discussion by shampaakter** [statichunt/geeky-hugo#2] [Show and tell]
Thanks for creating your site with Geeky Hugo. Let's share your website to inspire others. You can share your website link here, we will add it to the list.

**Comment by ziyooren** [comment]
I like Geeky Hugo, and have deployed 2 websites with it:

1. [theWindows12](https://www.thewindows12.com)
2. [CNELECAR](https://www.cnelecar.com)

Thanks for creating such a beautiful theme.
